src/pudl/workspace/datastore.py

```diff
@@ -200,11 +200,11 @@ class ZenodoDoiSettings(BaseSettings):
     eia_bulk_elec: ZenodoDoi = "10.5281/zenodo.11111208"
     epacamd_eia: ZenodoDoi = "10.5281/zenodo.7900974"
     epacems: ZenodoDoi = "10.5281/zenodo.11111203"
-    ferc1: ZenodoDoi = "10.5281/zenodo.8326634"
-    ferc2: ZenodoDoi = "10.5281/zenodo.8326697"
-    ferc6: ZenodoDoi = "10.5281/zenodo.8326696"
-    ferc60: ZenodoDoi = "10.5281/zenodo.8326695"
-    ferc714: ZenodoDoi = "10.5281/zenodo.8326694"
+    ferc1: ZenodoDoi = "10.5281/zenodo.12549172"
+    ferc2: ZenodoDoi = "10.5281/zenodo.11408175"
+    ferc6: ZenodoDoi = "10.5281/zenodo.11408164"
+    ferc60: ZenodoDoi = "10.5281/zenodo.12021231"
+    ferc714: ZenodoDoi = "10.5281/zenodo.12021228"
```

Comment:
I'm guessing these are the ones that include the 2023 data, and any other updates that have happened since we last updated the FERC archives. Do both the fast and full ETL actually work with the new archives if they're restricted to just extracting the 2021-2022 data?

Because using the new extractor necessitates using new input data, would it make sense for this update to be bundled with #3650 / #3710 instead of happening on its own?

Reply:
I've tested the fast ETL with no problems, and running the full ETL now. Also, I had talked to Christina about bundling them, and we decided to just get this in first, but I don't think either of us felt too strongly, and we don't get any new functionality until #3700 is complete anyway, so I'm fine holding off.

Reply:
This doesn't feel great to me since we know they go back and change / update old data without saying what changed, and we don't have an easy way to view or document the differences between the two outputs so I'm inclined to use this branch as the basis for the new 2023 data integration and bring in the new data and the new parsing that's required to read the new data at the same time, at which point it'll be more reasonable to expect there to be potentially significant changes to the outputs.

Reply:
That all sounds reasonable to me!